srcpkgs/tarpaulin/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'tarpaulin'
+pkgname=tarpaulin
+version=0.9.0
+revision=1
+archs="x86_64"
```

Comment:
Can you clarify why it is only for x86_64(-glibc)? Shouldn't it work at least traditionally on x86_64-musl?

Reply:
How would I enable this? Setting archs to "x86_64 x86_64-musl"?

Reply:
(Yes, it should work on musl)

Reply:
You can use wildcards `x86_64*`

Reply:
done :heavy_check_mark: